_docs/performance-tuning/drill-metastore/010-using-drill-metastore.md

```diff
@@ -10,6 +10,31 @@ The Metastore is a Beta feature; it is subject to change. We encourage you to tr
 Because the Metastore is in Beta, the SQL commands and Metastore formats may change in the next release.
 {% include startnote.html %}In Drill 1.17, this feature is supported for Parquet tables only and is disabled by default.{% include endnote.html %}
 
+## Drill Metastore introduction
+
+One of the main advantages of Drill is schema-on-read. But Drill can’t handle some cases with this approach, there are the issues related to Schema Evolution and Schema Changes.
+
+Significant benefits of schema-aware execution:
+
+ - At Planning time:
+    - Better scope for planning optimizations.
+    - Proper estimation of column widths since types are known, hence more accurate costing.
+    - Graceful early exit if certain data type validations fail.
+ - At Runtime:
+    - Avoids some cases with `SchemaChange` exceptions. All minor fragments will have a common understanding of the schema.
```

Comment:
Use noun to comply with other points (scope, estimation, exit): SchemaChange exception avoidance